frontend/lib/queries/LoginMutation.ts

```diff
@@ -21,6 +21,12 @@ export interface LoginMutation_login_errors {
   messages: string[];
 }
 
+export interface LoginMutation_login_session_onboardingStep1 {
+  name: string;
+  address: string;
+  aptNumber: string;
+}
+
```

Comment:
It's annoying how all our query TS files get bloated with replicas of our session type, but at least it's just type information that will get stripped out when compiling to JS.